src/State/ViewRequests/tests/reducers.test.js

```diff
@@ -0,0 +1,126 @@
+import viewRequestsReducer from '../reducers';
```

Comment:
Really like the nesting of describes and assertions, makes it really clear logically how to break it all down